### PR TITLE
Even more (small) improvements in the README

### PR DESCRIPTION
**Pull request by RaptaG**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
 
 ## Usage
 
-### FSB Skybox format
+### FSB skybox format
 
 The documentation on this format can be found [here](docs).
 
-### OptiFine Skybox format
+### OptiFine skybox format
```
Comment (AMereBagatelle) [inline README.md:22]: I believe `OptiFine` could be changed to `Optifine` here too, and throughout the document.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,38 @@
+<div align="center">
+<img src="https://raw.githubusercontent.com/AMereBagatelle/fabricskyboxes/1.19.x-dev/src/main/resources/assets/fabricskyboxes/fabricskyboxes_logo.png" width="150">
```
Comment (AMereBagatelle) [inline README.md]: Doesn't really add much, to be honest.  Doesn't look great in the rich diff either.  If a better logo is made in the future, I will consider adding it to the readme.

**Review by AMereBagatelle — CHANGES_REQUESTED** [review]
Thanks for your interest in making the README better!  Just a few notes.

File: README.md (hunk 2 above)
Comment (RaptaG) [reply README.md]: Ok I'll remove it. Just tell me if you want me to remove the alignment too

File: README.md (hunk 1 above)
Comment (RaptaG) [reply README.md:22]: But `OptiFine` is the correct capitalization

Comment (AMereBagatelle) [reply README.md:22]: Huh.  Thanks for letting me know!

**Review by AMereBagatelle — APPROVED** [review]

**Pull request merged**